google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/RpcViewConstants.java

```diff
@@ -124,4 +128,22 @@
               BIGTABLE_APP_PROFILE_ID,
               BIGTABLE_OP,
               BIGTABLE_STATUS));
+
+  static final View BIGTABLE_GFE_LATENCY_VIEW =
+      View.create(
+          View.Name.create("cloud.google.com/java/bigtable/gfe_latency"),
+          "GFE t4t7 latency in msecs",
```

Comment:
This description is meant for end users who don't really know what t4t7 latency means. Can you reword this a bit?